.github/workflows/gh-pages-deployment.yml

```diff
@@ -15,24 +14,26 @@ jobs:
       PUBLISH_DIR: ./build
 
     steps:
-    - name: Checkout Repository
-      uses: actions/checkout@v3
-
-    - uses: actions/setup-node@v3
-      with:
-        node-version: ${{ env.NODE_VERSION }}
-
-    - name: Install Dependencies
-      run: npm install
-
-    - name: Build
-      run: npm run build
-      env:
-        REACT_APP_BACKEND_URL: ${{ secrets.REACT_APP_BACKEND_URL }}
-        REACT_APP_BASENAME : ${{ secrets.REACT_APP_BASENAME }}
-
-    - name: Deploy to GitHub Pages
-      uses: peaceiris/actions-gh-pages@v3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: ${{ env.PUBLISH_DIR }}
+      - name: Checkout Repository
```

Comment:
Done. Although, @redat97 is already working on the workflow here https://github.com/ai-cfia/finesse-frontend/pull/45.

Reply:
yeah not sure I understand, you asked me to remove the gh-pages deployment this morning. I'll add the vercel deployment to this PR but lmk if that's not what you meant
https://github.com/ai-cfia/finesse-frontend/pull/45#discussion_r1415485044

Reply:
I got confused because you told me just before to remove the gh-pages deployment probably my mistake either way I updated it to vercel now and it's deployed to dev URL 
https://finesse-frontend-dev.vercel.app/

Reply:
@redat97 would please investigate why the app is not loading on either `https://ai-cfia.github.io/finesse-frontend/` or `https://finesse-frontend-dev.vercel.app/`. It worked prior to the use of the reusable build and deploy workflows and works locally.